### DISCUSSION TITLE
NAS Backup

### DISCUSSION THREAD

**Discussion by Scorpoon** [home-assistant/frontend#17343] [Other feature requests]
I just made a backup of home assistant to my nas and turned if off afterwards. 
It would be great to see the successfull backups in the backups menu even if my nas is offline. At the moment they only show up if my nas is online too.

**Comment by frenck** [comment]
This is not possible, as it cannot tell which backups are on your NAS if it is offline...

Additionally, this is not a frontend feature, but a backend feature request. Please use or community forum for that.

Thanks! 👍 

../Frenck
